programs/pkey/key_app.c

```diff
@@ -345,13 +344,6 @@ int main(int argc, char *argv[])
 
 cleanup:
 
-#if defined(MBEDTLS_ERROR_C)
```

Comment:
On second though we could print `ret` instead of its string translation. When we have that kind of translation back in TF-PSA-Crypto it will be easier to restore the current behavior.

Reply:
I have commented out the strerror lines in my latest commit. This will allow people to uncomment it when readded.